### PR TITLE
To Java

### PR DESCRIPTION
**Pull request by snh9858**

### CODE REVIEW

File: src/nodes/FunctionCallNode.java [hunk 1]
```diff
@@ -70,8 +70,19 @@ public String convertToJott() {
...
+        String funcName = id.token.getToken();
+        if (funcName.equals("print")) {
+            return "System.out.println(" + params.convertToJava(className) + "); "; // will only have 1 param
+        } else if (funcName.equals("concat")) {
+            String paramString = params.convertToJava(className);
+            String[] pList = paramString.split(",", 0); // there will always be 2 args, this has been validated
+            return pList[0] + " + " + pList[1] + " ";
+
+        } else if (funcName.equals("length")) {
+            return params.convertToJava(className) + ".length() "; // gg ez
```
Comment (law2460) [inline src/nodes/FunctionCallNode.java:83]: .length() should check hasSemiColon and append one if needed

File: src/nodes/FunctionDefParamsTypeNode.java [hunk 2]
```diff
@@ -55,8 +55,7 @@ public String convertToJott() {
 
     @Override
     public String convertToJava(String className) {
-        // TODO Auto-generated method stub
-        throw new UnsupportedOperationException("Unimplemented method 'convertToJava'");
+        return type.convertToJava(className) + " " + id.convertToJava(className);
```
Comment (law2460) [inline src/nodes/FunctionDefParamsTypeNode.java:58]:
```suggestion
        return "," + type.convertToJava(className) + " " + id.convertToJava(className);
```

Comment (law2460) [reply src/nodes/FunctionDefParamsTypeNode.java:58]: nevermind, you got the comma in ParamNode.java

File: src/nodes/FunctionCallNode.java (hunk 1 above)
Comment (snh9858) [reply src/nodes/FunctionCallNode.java]: fixed yippee 👍

**Pull request merged**